spec/system/kudos_spec.rb

```diff
@@ -152,6 +152,19 @@
       expect(Kudo.count).to eq 0
     end
 
+    it 'not being able to remove a kudo 5 minutes after it was sent' do
+      expect(Kudo.count).to eq 1
+      visit root_path
+
+      travel 6.minutes
+
+      visit root_path
+      click_link 'Destroy'
+
+      expect(page).not_to have_content 'Kudo was successfully destroyed.'
+      expect(Kudo.count).to eq 1
```

Comment:
I suggest time travel in the block, because, when we leave the block, time gets back to normal, and doesn't interfere with our other tests.

```ruby
travel 6.minutes do
  # some code
end
```

Reply:
I read about it. Therefore I have additional question -> if this block is needed in my situation? I added one test in 'it' block where I am only checking if clicking the destroy link 5 minutes after kudo creation provides to any action and notihing more. I guess that this travel 5.minutes only affects on this single 'it' block so if I don't test anything else in it I can skip travel block.

Reply:
You are right once again :)
I read somewhere someday that time travel has an impact on other tests, but I've tested it now, and time is going back to normal after the test with time travel ends :)

Reply:
Thanks! :D For certainty I tested it few times again :D